reaper-adm-extension/src/reaper_adm/pluginmain.cpp

```diff
@@ -281,12 +180,48 @@ extern "C" {
         btpActionName.c_str(),
         btpActionStrId.c_str(),
         [](admplug::ReaperAPI& api) {
-            if (!openDirectory(getToolsPath(api))) {
+            if (!ResourcePaths::openDirectory(ResourcePaths::getToolsPath(api))) {
                 api.ShowMessageBox("Failed to open directory.", "Browse Tools and Templates", 0);
             }
         }
     );
 
+    std::string ucDisableActionName("Disable update check on startup");
```

Comment:
Should these strings mention the EPS (is this the text that ends up in the REAPER action list?)

Reply:
Yep, good spot - didn't think of that